src/retention-data/retention.ts

```diff
@@ -0,0 +1,81 @@
+import { ActivityData, RetentionData } from './types';
+import { getISODateString, getDaysBetweenDates } from './utils';
+
+const getRedundant = (activity: ActivityData, previousActivities: ActivityData[]): void => {
```

Comment:
Can you explain the algorithm used to calculate redundant?

Reply:
Is it enough if we point to https://github.com/webhintio/hint/issues/3074?